bin/start-systemtest

```diff
@@ -0,0 +1,247 @@
+
+  echo "($(get_timestamp)) [${TEST_CASE}] Checking submitted tasks..."
+
+  # For each id of submitted tasks in this ${TEST_CASE}
+  #   While the task is processing
+  #     Get new task information by id and obtain the state
+  #     If the state is terminal ('archived' or 'failed'), then
+  #       Assert that the terminal state for the case of this task is as expected and skip to the next iteration
+  #     Else,
+  #       Sleep for X seconds (X is based on ${TEST_CASE}, to decrease access to the dispatcher's backend because some tests take longer than others)
+  for ID in $(echo "${TASK_IDS}" | jq ".[]" | tr -d \")
+  do
+    while true; do
+       local TASK=$(get_task_by_id ${ID})
+       local STATE=$(get_state "${TASK}")
+
+       echo "($(get_timestamp)) [${TEST_CASE}] Current task state [${ID}]: ${STATE}"       
+       if [ ${STATE} == 'archived' -o ${STATE} == 'failed' ]; then
+         echo "($(get_timestamp)) [${TEST_CASE}] Task [${ID}] has finished processing."
+         assert ${ID} ${TEST_CASE}
+         break
+       fi
+       echo "($(get_timestamp)) [${TEST_CASE}] Waiting ${TIMESLEEP} seconds to check again..."
+       sleep ${TIMESLEEP}
+    done
+  done
+}
+
+submit_task(){
+  local TEST_CASE=${1}
+  
+  bash ${SUBMIT_TASK_SCRIPT} ${USER_EMAIL} ${USER_PASSWORD} -7.413 -7.047 -37.314 -36.257 2015-06-23 2015-06-23 ${TEST_CASE} ${TEST_CASE} ${TEST_CASE} ${DISPATCHER_ADDRESS}
+}
+
+get_links(){
+  local ID=${1}
+
+  bash ${GET_TASK_LINKS_SCRIPT} ${USER_EMAIL} ${USER_PASSWORD} ${ID} ${DISPATCHER_ADDRESS}
+}
+
+get_name(){
+  # This function receives a string composed of <name>*<url> from a file processed from any SAPS task and returns only its name.
+  # Input example:
+  #   get_name out.log*<url>
+  # Output example:
+  #   out.log
+  local LINK=${1}
+  local NAME=$(echo ${LINK} | cut -d "*" -f1)
+
+  echo "${NAME}"
+}
+
+get_url(){
+  # This function receives a string composed of <name>*<url> from a file processed from any SAPS task and returns only its name.
+  # Input example:
+  #   get_name <name>*http://github.com/<anything>
+  # Output example:
+  #   http://github.com/<anything>
+  local LINK=${1}
+  local URL=$(echo ${LINK} | cut -d "*" -f2)
+
+  echo "${URL}"
+}
+
+get_md5sum(){
+  # This function receives a URL of a file processed from a SAPS task and calculates md5sum,
+  # without downloading the file, using the curl aligned with the md5sum command.
+  # Input example:
+  #   get_md5sum <url_file>
+  # Output example:
+  #   3de5943c4c9e5047ff2318829ad65f4d
+  local ULR=${1}
+  MD5SUM=$(curl --silent "${URL}" | md5sum | awk '{print $1}')
+
+  echo "${MD5SUM}"
+}
+
+check_md5sum_files(){
+  local ID=${1}
+  local TEST_CASE=${2}
+
+  echo "($(get_timestamp)) [${TEST_CASE}] Checking md5sum of task files [${ID}]..."
+  LINKS=$(get_links ${ID})
+
+  # This loop creates an iteration on top of a list created with '[<filename1>*<file_url1>, <filename2>*<file_url2>, ...]' (without quotes)
```

Comment:
based on the doc we added to line 192, what is the value of NAME_URL?

Reply:
It is a string that has the name and url concatenated by a *